msticpy/datamodel/entities/security_group.py

```diff
@@ -63,11 +63,19 @@ def description_str(self):
         """Return Entity Description."""
         return self.DistinguishedName or self.__class__.__name__
 
+    @property
```

Comment:
Also if we implemented this in the Entity class, we would only need to implement in entity sub-classes where it was returning something different to self.__class__.__name__